src/VisualStudio/CSharp/Impl/Options/Formatting/FormattingOptionPageControl.xaml.cs

```diff
@@ -29,6 +29,43 @@ public FormattingOptionPageControl(IServiceProvider serviceProvider) : base(serv
             BindToOption(FormatOnReturnCheckBox, FeatureOnOffOptions.AutoFormattingOnReturn, LanguageNames.CSharp);
             BindToOption(FormatOnPasteCheckBox, FeatureOnOffOptions.FormatOnPaste, LanguageNames.CSharp);
             SetNestedCheckboxesEnabled();
+
+            FormatDocumentSettingsGroupBox.Header = CSharpVSResources.Format_document_settings;
```

Comment:
this seems super strange.  Why isn't this handled in the xaml?  this isn't how we normally set these values elsewhere is it?

Reply:
This still isn't clear to me...

Reply:
all the string values in the FormattingOptionPageControl.xaml are replaced in the .xaml.cs file, I don't see why this one is different